user_settings.sql

```diff
@@ -0,0 +1,94 @@
+-- This is just a test (in draft) file so that Arit and I can work together using revision control
+-- We run the contents of this file in the rails dbconsole
+-- It will not be committed to the codebase, it will be moved into a data update script and this file will be deleted :)
+
+BEGIN TRANSACTION;
+WITH settings_data AS (
+  SELECT
```

Comment:
I did a cross check of these columns against the users_settings table and noticed that `permit_adjacent_sponsors` is missing. Can we add that please?

Reply:
Good catch thank you